team-manager/src/test/java/ua/knu/knudev/query/dsl/QueryDslTest.java

```diff
@@ -0,0 +1,137 @@
+package ua.knu.knudev.query.dsl;
```

Comment:
why do you need this, when you already have same tests in task manager>

Reply:
ok